bentoml/_internal/server/instruments.py

```diff
@@ -85,9 +125,127 @@ async def __call__(
             await response(scope, receive, send)
             return
 
-        service_version = (
-            self.bento_service.tag.version if self.bento_service.tag is not None else ""
+        endpoint = scope["path"]
+        START_TIME_VAR.set(default_timer())
+
+        async def wrapped_send(message: "ext.ASGIMessage") -> None:
+            if message["type"] == "http.response.start":
```

Comment:
Should we register time after the last byte of body is sent like in `access.py`?

Reply:
never thought about that